src/web/mjs/connectors/ComicTrail.mjs

```diff
@@ -39,13 +39,15 @@ export default class ComicTrail extends SpeedBinb {
         let request = new Request(this.url + manga.id, this.requestOptions);
         let data = await this.fetchDOM(request, this.queryChapters);
         return data.map(element => {
-            let part = "";
-            if (!/\d{4}-\d{1,2}-\d{1,2}/.test(element.children[1].firstElementChild.firstElementChild.children[1].textContent)){
-                part = ` ${element.children[1].firstElementChild.firstElementChild.children[1].textContent.trim()}`;
+            let id = "";
+            if (element.querySelector('a.button.is-read') != null){
+                id = `${element.querySelector('a.button.is-read').href}/`;
             }
+            let title = '';
+            element.querySelectorAll('p.has-text-weight-bold').forEach(element => title=title.concat(element.textContent.trim()));
```

Comment:
Do not use outer variable name `element` as scoped inner variable name
`element.querySelectorAll(...).forEach(el => el ...)`

Reply:
Should there be a space between the concatenated parts?
Maybe use `el => title += ' ' + el.textContent.trim()`
or `title = [...element.querySelectorAll('p.has-text-weight-bold')].join(' ')` instead